### PR TITLE
Add Github Actions build

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

**Testing done:**

https://github.com/gliptak/bottlerocket-update-operator/runs/1366499122?check_suite_focus=true

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Comment by gliptak** [comment]
@webern @jahkeup please review

**Review by etungsten — APPROVED** [review]
Thanks!

**Review by zmrow — APPROVED** [review]
:1st_place_medal:

**Review by jahkeup — APPROVED** [review]
Thanks @gliptak ! This is great!

I think we may want to build the container separately and improve the caching strategy for the SDK container image used. Though we can definitely follow up with that later on.

**Review by webern — APPROVED** [review]
Good idea, thank you!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+
```
Comment (webern) [inline .github/workflows/ci.yml:8]: We should probably run for pull requests.

```suggestion
      - develop
```

Comment (gliptak) [reply .github/workflows/ci.yml:8]: @webern the original already triggers for PRs see https://github.com/bottlerocket-os/bottlerocket-update-operator/pull/49

**Pull request merged**